review/npm.md

```diff
@@ -244,23 +244,8 @@ run.
         1. To run tests, run [`npm
            install-ci-test`](https://docs.npmjs.com/cli/v8/commands/npm-install-ci-test).
 
-    1. If you need to run a CLI package `tool-cli` from the registry:
-
-        1. Always declare it via a "wrapper" manifest file:
-           ```json
-           {
-             "name": "wrapper-tool-cli",
-             "version": "1.0.0",
-             "dependencies": {
-               "tool-cli": "4.1.1"
-             }
-           }
-           ```
-
-        1. ***Locally*** generate the corresponding lock file via `npm install
-           --package-lock-only`
-
-        1. Commit both files to the repository:
+    1. If you need to run a CLI package from the registry, ensure the package is a part of
+       your package.json and package-lock.json before reaching the automated environment.
```

Comment:
apps certainly need lockfiles (and this guidance is 100% true for apps), but it is far from universally agreed upon that *projects* should have one (even a dev-only lockfile).

Reply:
I removed the lockfile because while we agree here to refer to it (in other bullets) I thought it was sounding out of place here because no one actually goes and adds anything manually into a lockfile, and the prior bullets about a per-requisite to make use of a lockfile, make it appearance here redundant.

Reply:
LGTM. Thanks